tests/components/homematicip_cloud/test_hap.py

```diff
@@ -121,3 +150,84 @@
     await hap.async_reset()
 
     assert len(hass.config_entries.async_forward_entry_unload.mock_calls) == 8
+
+
+@patch("homematicip.aio.home.AsyncHome.__init__", Mock(return_value=None))
```

Comment:
Can we instead patch `AsyncHome` and spec it and if needed return a mock with the correct attributes?

This is too many decorators.

Reply:
I know,  but it worked.
To reduce the decorators i refactored the code a bit and added fixtures.